yarn-project/foundation/src/abi/note_selector.ts

```diff
@@ -14,7 +14,9 @@ export interface NoteSelector {
   _branding: 'NoteSelector';
 }
 
-/** A note selector is the first 4 bytes of the hash of a note signature. */
+/** A note selector is the first 4 bytes of the hash of a note signature
```

Comment:
This comment is now outdated. Also functions like `random` will now return invalid notes. We're going to remove this entire file soon as we move destructuring into aztec-nr, so there's no need to go all out, but it'd be good to at least superficially adjust the comments and stuff like `random`.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/10951/commits/74f25be2d1f5372a6b47697b7cdbe58820cb497b